recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 20736acea7dde31bbd3c6c6c3fbe922bedf242dd0b64bcd02a4557b4d566b427
+
+build:
+  number: 0
+  script: 
+    - git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw  # [unix]
+    - git clone https://github.com/kfrlib/kfr extern/kfr  # [unix]
+    - export LLVM_OPENMP="1"  # [osx]
+    - export NOISEFLOW_USE_CPP="1"  # [linux]
+    - export MACOSX_DEPLOYMENT_TARGET="10.13"  # [osx]
+    - {{ PYTHON }} -m pip install poetry-core@https://github.com/python-poetry/poetry-core/archive/refs/tags/1.6.1.zip
```

Comment:
Is the hardcode version something that can be avoided?

Reply:
Do you mean the `poetry-core v1.6.1`? 
I try to install the last version of poetry-core via `pip install poetry-core`, but it not works. I think I need to specify the version.